components/brave_sync/brave_profile_sync_service_impl.cc

```diff
@@ -201,8 +201,12 @@ void ClearDuplicatedNodes(ObjectIdToNodes* object_id_nodes,
   for (ObjectIdToNodes::iterator it_object_id = object_id_nodes->begin();
        it_object_id != object_id_nodes->end(); ++it_object_id) {
     const SortedNodes& nodes = it_object_id->second;
+    std::string object_id = it_object_id->first;
```

Comment:
This doesn't seem be to used

Reply:
yes thanks, this must be leftover from debug traces

Reply:
this is fixed